src/lib/components/MoveVerifyBadge.tsx

```diff
@@ -24,3 +30,24 @@ export const MoveVerifyBadge = ({ status }: MoveVerifyBadgeProps) => {
   }
   return undefined;
 };
+
+export const MoveVerifyBadge = ({
+  status,
+  hasTooltip = false,
+}: MoveVerifyBadgeProps) => {
+  const label = useMemo(() => {
+    if (status === MoveVerifyStatus.Outdated) {
+      return "This module was republished after verification.";
+    }
+    if (status === MoveVerifyStatus.Verified) {
+      return "This module has been verified.";
+    }
+    return undefined;
+  }, [status]);
```

Comment:
let's create a function in this file `getMoveVerifyBadgeLabel` and use it, so we don't need to have `useMemo`